From 62e572f078fb337ee3575f4ecde062609ee1e43f Mon Sep 17 00:00:00 2001
From: Niklas Cassel <cassel@kernel.org>
Date: Tue, 19 Aug 2025 15:32:52 +0200
Subject: [PATCH 01/17] do not let nvidia bootloader apply a bunch of overlays

We chainload our own bootloader, and do not want custom DTB overlays
to be applied by UEFI firmware when our bootloader boots.

We want our own bootloader to control which overlays that are applied,
if any.

Signed-off-by: Niklas Cassel <cassel@kernel.org>
---
 .../boot/dts/nvidia/tegra234-p3768-0000+p3767-0005.dts      | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/nvidia/tegra234-p3768-0000+p3767-0005.dts b/arch/arm64/boot/dts/nvidia/tegra234-p3768-0000+p3767-0005.dts
index 9f5e07012b87a..06281be17e876 100644
--- a/arch/arm64/boot/dts/nvidia/tegra234-p3768-0000+p3767-0005.dts
+++ b/arch/arm64/boot/dts/nvidia/tegra234-p3768-0000+p3767-0005.dts
@@ -16,4 +16,10 @@ hda@3510000 {
 	sound {
 		label = "NVIDIA Jetson Orin Nano APE";
 	};
+
+	firmware {
+		uefi {
+			firmware-media-overlays-applied;
+		};
+	};
 };
-- 
2.51.0

